modules/SofaGeneralLoader/SofaGeneralLoaderConfig.cmake.in

```diff
@@ -3,12 +3,15 @@
 @PACKAGE_GUARD@
 @PACKAGE_INIT@
 
-find_package(SofaFramework QUIET REQUIRED)
+find_package(Sofa.Component.IO.Mesh QUIET REQUIRED)
+find_package(Sofa.Component.IO.Misc QUIET REQUIRED)
 
-set(SOFAGENERALLOADER_HAVE_ZLIB @SOFAGENERALLOADER_HAVE_ZLIB@)
-if (SOFAGENERALLOADER_HAVE_ZLIB)
-    find_package(ZLIB QUIET REQUIRED)
+get_property(@PROJECT_NAME@_SENT_DEPRECATION_MESSAGE GLOBAL PROPERTY PROPERTY_@PROJECT_NAME@_SENT_DEPRECATION_MESSAGE SET)
+if(NOT @PROJECT_NAME@_SENT_DEPRECATION_MESSAGE)
+    message(WARNING "@PROJECT_NAME@ module is deprecated, use Sofa.Component.IO.Mesh and Sofa.Component.IO.Misc instead.")
```

Comment:
```suggestion
    message(WARNING "@PROJECT_NAME@ is deprecated. It will be removed at v23.06. Use Sofa.Component.IO.Mesh and Sofa.Component.IO.Misc instead.")
```